pkg/resources/common/validation.go

```diff
@@ -58,3 +61,50 @@ func ValidateRules(rules []rbacv1.PolicyRule, isNamespaced bool, fldPath *field.
 	}
 	return returnErr
 }
+
+var annotationsFieldPath = field.NewPath("metadata").Child("annotations")
+
+func CheckCreatorPrincipalName(userCache controllerv3.UserCache, obj metav1.Object) (*field.Error, error) {
```

Comment:
Since these functions are exported and in a common validation file, could you add a comment at the top of the functions describing them?

Reply:
Of course.